src/services/usersServices.js

```diff
@@ -0,0 +1,33 @@
+const { User } = require('./../models/User.model');
+
+const normalize = ({ id, name }) => {
+  return { id, name };
+};
+const getAll = () => {
+  return User.findAll({ raw: true });
+};
+
+const getById = (id) => {
+  return User.findByPk(id);
+};
+
+const create = (name) => {
+  return User.create({ name });
+};
+
+const remove = async (id) => {
+  await User.destroy({ where: { id } });
```

Comment:
Consider adding error handling for the `remove` function to manage potential exceptions, such as when the user does not exist.